packages/salesforcedx-utils/test/jest/cli/forceConfigGet.test.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  CliCommandExecutor,
+  CommandOutput,
+  ForceConfigGet
+} from '../../../src';
+import { JSON_FLAG } from '../../../src/cli/commandBuilder';
+
+describe('ForceOrgGet unit tests.', () => {
+  const fakeExecution = {
+    totally: 'fake'
+  };
+  // forceConfigGet expect and array with named keys which does not play nice in TS land
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const configMap: any = [
+    { key: 'config1', value: 'true' },
+    { key: 'defaultdevhubusername', value: 'test@test.com' },
+    { key: 'telemetryEnabled', value: 'false' }
+  ];
+
+  const resultObj = {
+    result: configMap
+  };
+  const fakeResult = JSON.stringify(resultObj);
+  const fakePath = '/here/is/a/fake/path';
+
+  let executeSpy: jest.SpyInstance;
+  let getCmdResultSpy: jest.SpyInstance;
+  beforeEach(() => {
+    executeSpy = jest
+      .spyOn(CliCommandExecutor.prototype, 'execute')
+      .mockReturnValue(fakeExecution as any);
+    getCmdResultSpy = jest
+      .spyOn(CommandOutput.prototype, 'getCmdResult')
+      .mockResolvedValue(fakeResult);
```

Comment:
Note: `mockResolved` value returns a promise wrapped results.  `mockReturnValue` is without promises.